includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -421,4 +417,96 @@ private function getPartnerSlot() {
 		}
 		return null;
 	}
+
+	private function getLogo() {
```

Comment:
I believe the name is self-explanatory. That's why we don't have PHPDoc for any method in this file but constructor.